tests/functional/test_fastfood_build_command.py

```diff
@@ -32,14 +37,14 @@
     }
   ]
 }
-""".strip()
+""").strip()
 
 
-class TestFastfoodBuildCommand(test_commands.TestFastfoodCommands):
+class TestFastfoodBuildCommand(TestFastfoodCommands):
 
     def test_fastfood_build_from_scratch(self):
 
-        build_config_file = tempfile.NamedTemporaryFile()
+        build_config_file = tempfile.NamedTemporaryFile(mode='w+')
```

Comment:
Good eye. I think `w+b` is the default mode for both python 2 & 3. The difference is that, if I leave the `b` (binary) in the open mode in python 3, I get `TypeError: 'str' does not support the buffer interface`, so it must be that the default `str` in python 2 is a bytearray, meaning it _does_ support the buffer interface, and the default `str` in python 3 is a unicode string, which does not support the buffer interface. So I opted to change the interface to non-binary rather than conditionally changing the input value depending on the python version. This seems ok, at least for this test, since the buildfiles should always be text.